src/page.py

```diff
@@ -26,16 +26,16 @@ def find_element(self, element):
             )
             return elem
         except TimeoutException:
-            logging.warning(f'The element {element[1]} has not been found'
+            logging.warning(f'The element {element[1]} has not been found '
                             f'by the {element[0]} locator.')
 
     def click_on_element(self, element):
         try:
-            WebDriverWait(self.driver, 3).until(
+            WebDriverWait(self.driver, 10).until(
```

Comment:
Why increase the timeout?

Reply:
To make it correspond with the find_element method, which has 10 seconds timeout.
So that the code looks a little more consistent.